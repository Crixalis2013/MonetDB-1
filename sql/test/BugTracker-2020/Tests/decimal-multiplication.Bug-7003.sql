select cast(0.1*0.1*0.1*0.1*0.1*0.1*0.1*0.1*0.1*0.1*0.1*0.1*0.1*0.1*0.1*0.1*0.1*0.1*0.1*0.1*0.1*0.1*0.1*0.1*0.1*0.1*0.1*0.1*0.1*0.1*0.1*0.1*0.1*0.1*0.1*0.1*0.1*0.1*0.1*0.1*0.1*0.1*0.1*0.1*0.1*0.1*0.1*0.1*0.1*0.1*0.1*0.1*0.1*0.1*0.1*0.1*0.1*0.1*0.1*0.1*0.1*0.1*0.1*0.1*0.1*0.1*0.1*0.1*0.1*0.1*0.1*0.1*0.1*0.1*0.1*0.1*0.1*0.1*0.1*0.1*0.1*0.1*0.1*0.1*0.1*0.1 as decimal(16,15));
-- 0.0000000000000

select 0.1+0.1+0.1+0.1+0.1+0.1+0.1+0.1+0.1+0.1+0.1+0.1+0.1+0.1+0.1+0.1+0.1+0.1+0.1+0.1+0.1+0.1+0.1+0.1+0.1+0.1+0.1+0.1+0.1+0.1+0.1+0.1+0.1+0.1+0.1+0.1+0.1+0.1+0.1+0.1+0.1+0.1+0.1+0.1+0.1+0.1+0.1+0.1+0.1+0.1+0.1+0.1+0.1+0.1+0.1+0.1+0.1+0.1+0.1+0.1+0.1+0.1+0.1+0.1+0.1+0.1+0.1+0.1+0.1+0.1+0.1+0.1+0.1+0.1+0.1+0.1+0.1+0.1+0.1+0.1+0.1+0.1+0.1+0.1+0.1+0.1;
-- 8.6

select cast(0.1 as double)*0.1*0.1*0.1*0.1*0.1*0.1*0.1*0.1*0.1*0.1*0.1*0.1*0.1*0.1*0.1*0.1*0.1*0.1*0.1*0.1*0.1*0.1*0.1*0.1*0.1*0.1*0.1*0.1*0.1*0.1*0.1*0.1*0.1*0.1*0.1*0.1*0.1*0.1*0.1*0.1*0.1*0.1*0.1*0.1*0.1*0.1*0.1*0.1*0.1*0.1*0.1*0.1*0.1*0.1*0.1*0.1*0.1*0.1*0.1*0.1*0.1*0.1*0.1*0.1*0.1*0.1*0.1*0.1*0.1*0.1*0.1*0.1*0.1*0.1*0.1*0.1*0.1*0.1*0.1*0.1*0.1*0.1*0.1*0.1*0.1;
-- 1.0e-86

select power(0.1, 86);
-- 1.0e-86

