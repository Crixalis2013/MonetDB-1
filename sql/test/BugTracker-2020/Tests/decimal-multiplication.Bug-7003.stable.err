stderr of test 'decimal-multiplication.Bug-7003` in directory 'sql/test/BugTracker-2020` itself:


# 21:08:04 >  
# 21:08:04 >  "mclient" "-lsql" "-ftest" "-tnone" "-Eutf-8" "-i" "-e" "--host=/var/tmp/mtest-13109" "--port=35110"
# 21:08:04 >  


# 21:08:04 >  
# 21:08:04 >  "Done."
# 21:08:04 >  

