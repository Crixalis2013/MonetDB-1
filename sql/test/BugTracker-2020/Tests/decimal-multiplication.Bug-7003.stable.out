stdout of test 'decimal-multiplication.Bug-7003` in directory 'sql/test/BugTracker-2020` itself:


# 21:08:04 >  
# 21:08:04 >  "mclient" "-lsql" "-ftest" "-tnone" "-Eutf-8" "-i" "-e" "--host=/var/tmp/mtest-13109" "--port=35110"
# 21:08:04 >  

#select cast(0.1*0.1*0.1*0.1*0.1*0.1*0.1*0.1*0.1*0.1*0.1*0.1*0.1*0.1*0.1*0.1*0.1*0.1*0.1*0.1*0.1*0.1*0.1*0.1*0.1*0.1*0.1*0.1*0.1*0.1*0.1*0.1*0.1*0.1*0.1*0.1*0.1*0.1*0.1*0.1*0.1*0.1*0.1*0.1*0.1*0.1*0.1*0.1*0.1*0.1*0.1*0.1*0.1*0.1*0.1*0.1*0.1*0.1*0.1*0.1*0.1*0.1*0.1*0.1*0.1*0.1*0.1*0.1*0.1*0.1*0.1*0.1*0.1*0.1*0.1*0.1*0.1*0.1*0.1*0.1*0.1*0.1*0.1*0.1*0.1*0.1 as decimal(14,13));
% .%1 # table_name
% %1 # name
% decimal # type
% 18 # length
[ 0.000000000000000	]
#select 0.1+0.1+0.1+0.1+0.1+0.1+0.1+0.1+0.1+0.1+0.1+0.1+0.1+0.1+0.1+0.1+0.1+0.1+0.1+0.1+0.1+0.1+0.1+0.1+0.1+0.1+0.1+0.1+0.1+0.1+0.1+0.1+0.1+0.1+0.1+0.1+0.1+0.1+0.1+0.1+0.1+0.1+0.1+0.1+0.1+0.1+0.1+0.1+0.1+0.1+0.1+0.1+0.1+0.1+0.1+0.1+0.1+0.1+0.1+0.1+0.1+0.1+0.1+0.1+0.1+0.1+0.1+0.1+0.1+0.1+0.1+0.1+0.1+0.1+0.1+0.1+0.1+0.1+0.1+0.1+0.1+0.1+0.1+0.1+0.1+0.1;
% .%2 # table_name
% %2 # name
% decimal # type
% 4 # length
[ 8.6	]
#select cast(0.1 as double)*0.1*0.1*0.1*0.1*0.1*0.1*0.1*0.1*0.1*0.1*0.1*0.1*0.1*0.1*0.1*0.1*0.1*0.1*0.1*0.1*0.1*0.1*0.1*0.1*0.1*0.1*0.1*0.1*0.1*0.1*0.1*0.1*0.1*0.1*0.1*0.1*0.1*0.1*0.1*0.1*0.1*0.1*0.1*0.1*0.1*0.1*0.1*0.1*0.1*0.1*0.1*0.1*0.1*0.1*0.1*0.1*0.1*0.1*0.1*0.1*0.1*0.1*0.1*0.1*0.1*0.1*0.1*0.1*0.1*0.1*0.1*0.1*0.1*0.1*0.1*0.1*0.1*0.1*0.1*0.1*0.1*0.1*0.1*0.1*0.1;
% .%2 # table_name
% %2 # name
% double # type
% 24 # length
[ 1e-86	]
#select power(0.1, 86);
% .%2 # table_name
% %2 # name
% double # type
% 24 # length
[ 1e-86	]

# 21:08:04 >  
# 21:08:04 >  "Done."
# 21:08:04 >  

